astropy/io/fits/convenience.py

```diff
@@ -602,7 +602,10 @@ def append(filename, data, header=None, checksum=False, verify=True, **kwargs):
 
     kwargs
         Any additional keyword arguments to be passed to
-        `astropy.io.fits.open`.
+        `astropy.io.fits.open`. Please note that scale_back is not accepted
+        as an argument for a file that does not exist originally with kwargs to
+        be passed to `astropy.io.fits.writeto`. In the latter scenario, the
+        only option that can be used is output_verify.
```

Comment:
This is not the expected change. There is no reason to mention scale_back in particular, what is needed is to know to which function the kwargs are passed.

Reply:
Okay, I will delete the reference about `scale_back` then...